x-pack/legacy/plugins/apm/cypress/cypress/plugins/index.js

```diff
@@ -19,36 +19,10 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies
 const wp = require('@cypress/webpack-preprocessor');
-const fs = require('fs');
 
 module.exports = on => {
-  // add typescript support
   const options = {
-    webpackOptions: {
-      resolve: {
-        extensions: ['.ts', '.tsx', '.js']
-      },
-      module: {
-        rules: [
-          {
-            test: /\.tsx?$/,
-            loader: 'ts-loader',
-            options: { transpileOnly: true }
-          }
-        ]
-      }
-    }
+    webpackOptions: require('../webpack.config.js')
   };
   on('file:preprocessor', wp(options));
-
-  // readFileMaybe
-  on('task', {
```

Comment:
Not sure what it's the purpose of this function, maybe to read the `snapshots.js` file? If so, Cucumber + Cypress provides a `fixtures` dir to store JSON files as data for the tests.

Reply:
Yep, it's for the snapshots IIRC.